scheduler/util_test.go

```diff
@@ -972,6 +973,35 @@ func TestInplaceUpdate_Success(t *testing.T) {
 	}
 }
 
+func TestInplaceUpdate_WildcardDatacenters(t *testing.T) {
+	ci.Parallel(t)
+
+	state, ctx := testContext(t)
```

Comment:
nit: this variable name collides with an import

Reply:
Good catch, that always annoys me but I copy-pasted a lot of the setup code in this test :grinning: Fixed!